libs/input-mapping/tests/Functional/DownloadTablesWorkspaceBigQueryTest.php

```diff
@@ -74,9 +77,12 @@ public function testTablesBigQueryBackend(): void
         self::assertTrue($this->testHandler->hasInfoThatContains('Processed 1 workspace exports.'));
         // test that the clone jobs are merged into a single one
         sleep(2);
-        $jobs = $this->clientWrapper->getTableAndFileStorageClient()->listJobs(['limit' => 20]);
+        $jobs = $this->clientWrapper->getTableAndFileStorageClient()->listJobs();
         $params = null;
         foreach ($jobs as $job) {
+            if ($runId !== $job['runId']) {
+                continue;
+            }
```

Comment:
jako spíš by bylo lepší z toho udělat array_filter, ale otázka je, jestli to najítí přes workspaceLoad stačí a neměl by se tam přidat nějaká další informace pro to filtrování - právě to runid?

Reply:
Udelal jsem array_filter